etl/extractor/mainnet_impl.go

```diff
@@ -34,6 +37,107 @@ func NewMainNetExtractor(batchSize uint32, exporterClient exporter.RecordExporte
 }
 
 func (m *MainNetExtractor) GetJetDrops(ctx context.Context) <-chan *types.PlatformJetDrops {
+	return m.mainJetDropsChan
+}
+
+func (m *MainNetExtractor) LoadJetDrops(ctx context.Context, fromPulseNumber int, toPulseNumber int) error {
+	if fromPulseNumber < 0 {
+		return errors.New("fromPulseNumber cannot be negative")
+	}
+	if toPulseNumber < 1 {
+		return errors.New("toPulseNumber cannot be less than 1")
+	}
+	if fromPulseNumber > toPulseNumber {
+		return errors.New("fromPulseNumber cannot be greater than toPulseNumber")
+	}
+	unsignedToPulseNumber := uint32(toPulseNumber)
```

Comment:
Why `unsigned` ?

Reply:
because of receivedPulseNumber is uint32. so we need to convert to unit once